hydrus/core/HydrusText.py

```diff
@@ -96,11 +96,33 @@ def LooksLikeHTML( file_data ):
     
     if isinstance( file_data, bytes ):
         
-        search_elements = ( b'<html', b'<HTML', b'<title', b'<TITLE' )
+        search_elements = ( b'<html', b'<HTML', b'<!DOCTYPE html', b'<!DOCTYPE HTML' )
```

Comment:
SVG files can also have `title` elements so that had to be removed. I also added the doctype check here for consistency.

Reply:
hello I am having a conversation on github.com